### ISSUE TITLE
Mid

### ISSUE THREAD

**Issue by mobinsamin** [mobinsamin/hasan#1]
import gradio as gr
from diffusers import StableDiffusionInpaintPipeline
import torch, os

# تنظیم مسیر مدل (فایل .ckpt یا .safetensors را داخل models/ قرار دهید)
MODEL_PATH = "/sdcard/MedDiffusion/models/meddiff_lite.safetensors"
OUTPUT_DIR = "/sdcard/MedDiffusion/outputs"
os.makedirs(OUTPUT_DIR, exist_ok=True)

# بارگذاری مدل سبک مخصوص inpainting پزشکی
pipe = StableDiffusionInpaintPipeline.from_single_file(
    MODEL_PATH,
    torch_dtype=torch.float16
).to("cpu")  # اجرای CPU روی تبلت

# تابع بازسازی پزشکی

def reconstruct(image, mask, prompt):
    result = pipe(prompt=prompt, image=image, mask_image=mask).images[0]
    save_path = os.path.join(OUTPUT_DIR, "reconstructed.jpg")
    result.save(save_path)
    return result, save_path

# رابط کاربری
interface = gr.Interface(
    fn=reconstruct,
    inputs=[
        gr.Image(label="عکس اصلی (واقعی)", type="pil"),
        gr.Image(label="ماسک ناحیه آسیب‌دیده", type="pil"),
        gr.Textbox(label="توضیح کوتاه (مثلاً: بازسازی پوست سالم قبل از آسیب)")
    ],
    outputs=["image", "text"],
    title="MedDiffusion Lite – Reconstruction"
)

interface.launch(server_name="0.0.0.0", server_port=7860)
